src/syrupy/location.py

```diff
@@ -5,8 +5,10 @@
     Optional,
 )
 
+from syrupy.constants import PYTEST_NODE_SEP
 
-class TestLocation:
+
+class PyTestLocation:
```

Comment:
Changed the name because pytest was complaining when attempting to collect the class as a test.